app/src/Components/Project/ProjectAttachments.tsx

```diff
@@ -17,10 +21,20 @@ const AttachmentDetails = styled.Text`
   font-size: 16px;
   margin-left: 12px;
 `
-
-const ProjectAttachments: FC<ProjectAttachmentsProps> = ({ icon, details }) => {
+/**
+ * Displays the attachment button
+ * @param {string} url of project scope url
+ * @returns {ReactElement} project attachment button and text
+ */
+const ProjectAttachments: FC<ProjectAttachmentsProps> = ({
+  icon,
+  details,
+  url,
+}) => {
+  // Project Attachments will not be displayed if URL is null or not a validate format
+  if (!url || !url.match('drive.google.com')) return null
   return (
-    <AttachmentButton onPress={underDevelopmentAlert}>
+    <AttachmentButton onPress={() => navigate('ProjectScope', { url: url })}>
```

Comment:
Optional, but if you want to simplify this line slightly you can change `url: url` to just `url`  (This is a little shortcut in Javascript, for when the property and the variable you're setting it to are both named the same.)